### PR TITLE
SHARD-254: remove string matching check to avoid string case errors

### PR DESCRIPTION
**Pull request by arhamj**
https://linear.app/shm/issue/GOLD-280/

If not fixed, the side effect is that keys are case-sensitive.

### CODE REVIEW

**Review by abdulazeem-tk4vr — DISMISSED** [review]

**Comment by chrypnotoad** [comment]
/ask "What is getting deleted in this PR"

**Comment by chrypnotoad** [comment]
@CodiumAI-Agent /improve

**Review by aniketdivekar — DISMISSED** [review]

File: src/setup/helpers.ts [hunk 1]
```diff
@@ -139,11 +140,12 @@ export function verifyMultiSigs(
       !seen.has(sigs[i].owner) &&
       allowedPubkeys[sigs[i].owner] &&
       allowedPubkeys[sigs[i].owner] >= requiredSecurityLevel &&
-      ethers.verifyMessage(payload_hash, sigs[i].sig) === sigs[i].owner
+      ethers.verifyMessage(payload_hash, sigs[i].sig)
```
Comment (chrypnotoad) [inline src/setup/helpers.ts]: Doesn't this remove the check that the signature is from the owner entirely?
The issue is that this `===` is case sensitive and it should not be. I don't think the removing the check is correct

Comment (jintukumardas) [reply src/setup/helpers.ts]: Arham is OOO. I've updated it. Thanks!

**Review by aniketdivekar — DISMISSED** [review]

**Comment by jintukumardas** [comment]
Making additional modifications to the code. Saving it as a draft.

**Comment by jintukumardas** [comment]
Ready for review

**Review by aniketdivekar — APPROVED** [review]

**Review by abdulazeem-tk4vr — APPROVED** [review]

**Review by chrypnotoad — APPROVED** [review]

**Pull request merged**